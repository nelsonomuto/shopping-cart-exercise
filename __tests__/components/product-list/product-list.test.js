import { shallow } from "enzyme"
import React from "react"
import ProductItem from "../../../src/components/product/product-item"
import ProductList from "../../../src/components/product/product-list"

describe("<ProductList />", () => {
  it("renders product list", () => {
    const component = shallow(<ProductList />)
    expect(component.find(ProductItem)).toHaveLength(1)
    expect(component).toMatchSnapshot()
  })
})
