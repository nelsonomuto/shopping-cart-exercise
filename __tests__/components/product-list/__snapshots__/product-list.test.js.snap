// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductList /> renders product list 1`] = `
<div
  className="product-list grid-x grid-margin-y"
>
  <ProductItem
    image="fake-image-url"
    key="fake-sku"
    price={999.99}
    sku="fake-sku"
    title="fake-title"
  />
</div>
`;
